src/components/toplayer/_toplayer.scss

```diff
@@ -74,6 +74,29 @@ $includeHtml: false !default;
       }
     }
 
+    &--small-spaced {
+      @include sgBreakpoint(small-only) {
+        margin-top: gutter(4);
+        max-width: gutter(14);
+        margin-left: gutter(1);
+        margin-right: gutter(1);
+      }
+    }
+
+    &--splash-screen {
+      overflow: hidden;
+      margin: gutter(1 / 2);
+      // sass-lint:disable no-duplicate-properties
+      min-height: 100%;
+      min-height: calc(100% - #{gutter(1)});
```

Comment:
I'd add comment here to change it when they'll fix it in sass (or just to have reference): https://github.com/sass/sass/issues/818